openwisp_notifications/static/openwisp-notifications/js/notifications.js

```diff
@@ -324,7 +338,54 @@ function notificationWidget($) {
         if (elem.hasClass('unread')) {
             markNotificationRead(elem.get(0));
         }
-        window.location = elem.data('location');
+
+        var container = document.querySelector('#container');
```

Comment:
Why are we mixing vanilla JS with jQuery? If the code uses jQuery, we should be consistent, we can't start doing the same operations in different ways because it makes maintenance more costly.

Reply:
Updated